src/main/io/osd.c

```diff
@@ -1628,6 +1670,16 @@ static void osdShowStats(void)
         itoa(mAhDrawn, buff, 10);
         strcat(buff, "\x07");
         displayWrite(osdDisplayPort, statValuesX, top++, buff);
+
+        if (STATE(GPS_FIX)) {
```

Comment:
Should we check here? Isn't non-zero `totalDistance` a sign that we did have GPS at least some of the trip?